apps/web/components/eventtype/AvailabilityTab.tsx

```diff
@@ -157,6 +157,7 @@ const EventTypeScheduleDetails = () => {
         </span>
         <Button
           href={`/availability/${schedule?.schedule.id}`}
+          disabled={isLoading}
```

Comment:
LGTM however,  - I think it would be best to introduce IsInitalLoading here 

So we dont disable this button on every refresh 

https://tanstack.com/query/v4/docs/react/guides/disabling-queries#isinitialloading

Reply:
Awesome suggestion, this only became available in `@tanstack/query` > 4.8.0, for context we are currently on 4.3.4 based on the `yarn.lock`. Would we also like to upgrade the package as part of this change? 🤔

Reply:
Nah; this is good for now 🙏 - I was unaware this was so new!

Reply:
> Awesome suggestion, this only became available in `@tanstack/query` > 4.8.0, for context we are currently on 4.3.4 based on the `yarn.lock`. Would we also like to upgrade the package as part of this change? 🤔

I had no idea thank you :)